src/libraries/System.Utf8String.Experimental/src/System/Utf8String.Portable.cs

```diff
@@ -0,0 +1,231 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Text.Unicode;
+
+namespace System
+{
+    public sealed partial class Utf8String
+    {
+        private readonly byte[] _bytes;
+
+        /// <summary>
+        /// Returns the length (in UTF-8 code units, or <see cref="byte"/>s) of this instance.
+        /// </summary>
+        public int Length => _bytes.Length == 0 ? 0 : _bytes.Length - 1; // -1 because the bytes are always null-terminated
```

Comment:
Will *_bytes* ever be empty? Thought it'd always have length ≥1 due to the null terminator.

Reply:
No it shouldn't. This was left over from an earlier implementation. Fixed.